frontend/pages/mountains.py

```
@@ -0,0 +1,110 @@
+    if df.empty:
+        return None
+
+    m = folium.Map(location=[df["Latitude"].mean(), df["Longitude"].mean()])
+    m.fit_bounds([
+        [df["Latitude"].min(), df["Longitude"].min()],
+        [df["Latitude"].max(), df["Longitude"].max()]
+    ])
+    Fullscreen().add_to(m)
+
+    for _, row in df.iterrows():
+        color = "green" if row["climbed"] == True else "red"
```

Ruff (0.15.5)

[error] Avoid equality comparisons to `True`; use `row["climbed"]:` for truth checks

Replace with `row["climbed"]`

(E712)